electron/index.js

```diff
@@ -153,61 +155,80 @@ ipcMain.on("exit-app", (event, flag) => {
 });
 
 function performFsJob(job) {
-    const fname = path.join(storePath, job.filename);
+    let fname = path.join(storePath, job.filename);
+    if (job.mods) fname = path.join(modsPath, job.filename);
```

Comment:
wouldn't this allow passing `../../../something.txt` as a filename?

Reply:
Yes forgot that

Reply:
Fixed this